src/WebJobs.Extensions.DurableTask/LinuxAppServiceLogger.cs

```diff
@@ -59,8 +58,6 @@ internal class LinuxAppServiceLogger
             this.roleInstance = JToken.FromObject("App-" + containerName);
             this.tenant = JToken.FromObject(tenant);
 
-            this.sourceMoniker = JToken.FromObject(
-                string.IsNullOrEmpty(stampName) ? string.Empty : "L" + stampName.Replace("-", "").ToUpperInvariant());
```

Comment:
Is it correct that you were never actually using this value? I don't see it referenced anywhere else. If so, I guess my next question is whether there is an opportunity to make the test more robust.

Reply:
@cgillum, sorry this is just a test PR that I was running because, for some reason, the Azure Pipelines CI stopped running our test. I should have made this a draft, will take a mental note. Didn't mean to be reviewed

Reply:
But, to answer your questions:
I removed the usage of `this.sourceMoniker` on a previous PR, that's why you did not see that here.
That PR is this one:https://github.com/Azure/azure-functions-durable-extension/pull/1482

My test should have caught on to the fact that `this.sourceMoniker` was not getting logged (and in fact they do) but Azure Pipelines is, somehow, no longer automatically running the tests for new PRs. The tests you're currently seeing that are running for this PR were manually triggered.

Reply:
In other words, the Azure Pipelines CI is acting up.
The result is that we're currently getting "green checkmarks" in al our PRs but those are deceiving because the tests are not getting run. For instance:
here: https://github.com/Azure/azure-functions-durable-extension/pull/1513
here: https://github.com/Azure/azure-functions-durable-extension/pull/1514

So, I'm manually running this test and, if it passes, I'm going to propagate this change to dev and master because, otherwise, we have a failing build. After that, I'll investigate what's going on with the CI.

I found out that the CI is not running our tests as I was inspecting the artifacts page of Azure  Pipelines, so that's why we did not catch it when promoting dev to master. I'm looking to fix that now